dune-project

```diff
@@ -1,2 +1,102 @@
-(lang dune 1.7)
+(lang dune 1.11)
+
 (using fmt 1.1)
+
+(name pgx)
+
+(generate_opam_files true)
+
+(license "LGPL-2 with OCaml linking exception")
+
+(maintainers "Arena Developers <silver-snakes@arena.io>")
+
+(authors "Arena Developers <silver-snakes@arena.io>")
+
+(source
+ (github arenadotio/pgx))
+
+(package
+ (name pgx)
+ (synopsis "Pure-OCaml PostgreSQL client library")
+ (description
+   "PGX is a pure-OCaml PostgreSQL client library, supporting Async, LWT, or synchronous operations.")
+ (depends
+  (alcotest
+   (and
+    :with-test
+    (>= 1.0.0)))
+  (base64
+   (and
+    :with-test
+    (>= 3.0.0)))
+  (bisect_ppx
+   (and
+    :dev
+    (>= 2.0.0)))
+  (dune
+   (>= 1.11))
+  ipaddr
+  (ocaml
+   (>= 4.08))
+  (ppx_custom_printf
+   (>= v0.13.0))
+  (ppx_sexp_conv
+   (>= v0.13.0))
+  re
+  (sexplib0
+   (>= v0.13.0))
+  uuidm))
+
+(package
+ (name pgx_unix)
+ (synopsis
+   "PGX using the standard library's Unix module for IO (synchronous)")
+ (depends
+  (ocaml
+   (>= 4.08))
```

Comment:
Won't it pick this up automatically from the pgx=version dep?

Reply:
It should. I think I forgot to remove this from here after i was done verifying that everything works. But it probably doesn't hurt to leave it in here now since we have a single file to touch during updates?